src/components/RegistrationModal.tsx

```diff
@@ -74,7 +74,7 @@ const RegistrationModal = ({
         walletAddress,
         formValues.handle
       );
-      onIdResolved(core.identifiers.convertDSNPUserURIToDSNPUserId(userURI));
+      onIdResolved(core.identifiers.convertToDSNPUserId(userURI).toString());
```

Comment:
This seems needlessly complicated since userURI is already a string.  We should have an SDK function that just strips the scheme prefix and returns the ID as a string.  Not for this PR obviously though.

Reply:
Totally agreed.